### PR TITLE
custom-app revisions

### PR DESCRIPTION
**Pull request by jerrycowell96**

### CODE REVIEW

File: website/docs/brain/custom-applications.mdx [hunk 1]
```diff
@@ -69,6 +69,10 @@ ssh-keygen # create ~/.ssh/ folder
 
 3. Create a file called `config` (no extension) in `~/.ssh`
 
+```bash
+cd ~/.ssh
+gedit config
```
Comment (edgarriba) [inline website/docs/brain/custom-applications.mdx:74]: just `gedit ~/.shh/config` should work

eventhough i like more vim :)

**Review by edgarriba — APPROVED** [review]
LGMT

File: website/docs/brain/custom-applications.mdx [hunk 2]
```diff
@@ -306,3 +314,34 @@ git config --global user.name "YOUR-USERNAME"
...
+```bash
+ln -s ~/workspace/amiga/apps/camera/ ~/apps/camera
+```
+
+toggle off apps screen, then back and Camera Dev should appear. Repeat for apps you want to see.
+
+**Option 2 (env variable)**
+```bash
+export FARM_NG_APPS_PATH=/data/home/amiga/workspace/amiga/apps/
+```
```
Comment (Hackerman342) [inline website/docs/brain/custom-applications.mdx:347]: These are internal docs , for SEP not dev kit

Comment (jerrycowell96) [reply website/docs/brain/custom-applications.mdx:347]: Oh okay.

Comment (Hackerman342) [reply website/docs/brain/custom-applications.mdx:347]: The whole ## Run workspace version of launcher section I mean

**Pull request merged**